### PR TITLE
docs: Fix `ssh` test command

### PR DESCRIPTION
**Pull request by tyler36**
## The Issue

The current command generates an error:

```shell
❯ ssh git@github.com git@github.com
Invalid command: git@github.com
You appear to be using ssh to clone a git:// URL.
```

Confirmed issues in the following terminals:

- PowerShell 7.4.6
- Windows Powershell 5.1.22621.4391
- Git bash 2.47.0.windows.2

## How This PR Solves The Issue

Update the command.
Now generates the following:

```
❯  ssh git@github.com
Enter passphrase for key '/c/Users/XXXXXX/.ssh/id_ed25519':
Hi XXXXXX! You've successfully authenticated, but GitHub does not provide shell access.
```

### CODE REVIEW

**Comment by tyler36** [comment]
Looking at https://docs.github.com/en/authentication/connecting-to-github-with-ssh/testing-your-ssh-connection, they recommend:

```
ssh -T git@github.com
```

File: src/content/blog/windows-ddev-setup.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Two recent Windows machines I set up were the new ARM64/Qualcomm/CoPilot variety
     * On ARM64, `choco uninstall -y mkcert gsudo` so that the DDEV installer can get the native versions of each of these.
     * On ARM64, install the Windows-side DDEV from the installer in the [DDEV releases](https://github.com/ddev/ddev/releases). We'll probably discontinue documenting the Chocolatey install technique in the future.)
 11. Install and test the fantastic [1Password ssh agent](https://developer.1password.com/docs/ssh/agent/).
-12. On Windows PowerShell `ssh git@github.com git@github.com` to verify that the 1Password SSH agent is working. If it says "PTY Allocation Failed", just hit `<RETURN>` and ignore it. You should get the confirmation message from GitHub.
+12. On Windows PowerShell `ssh -T git@github.com` to verify that the 1Password SSH agent is working. If it says "PTY Allocation Failed", just hit `<RETURN>` and ignore it. You should get the confirmation message from GitHub.
```
Comment (rfay) [inline src/content/blog/windows-ddev-setup.md:43]: What does the `-T` do?

**Review by rfay — APPROVED** [review]
Yay, @tyler36 is back! Life can move forward again!

File: src/content/blog/windows-ddev-setup.md (hunk 1 above)
Comment (tyler36) [reply src/content/blog/windows-ddev-setup.md:43]: Can seem to find much but https://explainshell.com/explain?cmd=ssh%20-t says:

> -T Force pseudo-tty allocation.

https://stackoverflow.com/questions/42505339/why-use-t-with-ssh has some more explaination.

Without the `-T`, the command runs alot slower and I get an `PTY allocation request failed on channel 0` error.

IIRC, most SSH debugging tutorials including `-T`.

**Comment by rfay** [comment]
Thanks, will pull when you're ready, thanks for -T ! Great to have you back.

**Comment by tyler36** [comment]
Ready when you are.

**Comment by rfay** [comment]
Would appreciate any other improvements on that blog!

**Pull request merged**